Resources/Prototypes/Reagents/fun.yml

```
@@ -236,22 +236,84 @@
             Cold: 0.05
       - !type:AdjustTemperature
         conditions:
-        - !type:Temperature
-          min: 160.15
+        - !type:Temperature #Start Harmony Change
+          min: 60.15
+          max: 300.0
```

```suggestion
#Start Harmony Change
        - !type:Temperature
          #min: 160.15
          min: 60.15
          max: 300.0
```
You should comment the value that got changed as well.